buildSrc/src/main/kotlin/configurations/Languages.kt

```diff
@@ -18,22 +18,12 @@ object Languages {
     fun Project.attachRemoteRepositories() {
         repositories {
             mavenCentral()
+            maven(url = "https://dl.bintray.com/thewca/tnoodle-lib")
```

Comment:
Will eventually become "fully" public, i.e. mirrored to the central repositories that don't require a custom link to our specific Bintray.

Setting up these mirrors has been initiated, but it takes a lot of processing time (up to 48 hours initially) because of how Maven works.

_Technically_, the package already is public, you just need to specifically point your build tool of choice to this specific (but public!) URL

Reply:
Very naive question: in case bintray goes down, is there a way to make it point to a local copy of tnoodle-lib?

Reply:
Yes, that is easily possible by adding a local `path/to/repo` replication here. However, I highly doubt Bintray will go down for > 30 seconds at once that easily